src/Coalesce.React/common-components/src/js/templateController.js

```diff
@@ -80,8 +80,8 @@ export function loadTemplateByName(name, source, version)
 
 export function registerTemplate(key)
 {
-  return fetch(`${karafRootAddr}/templates/${key}`, {
-      method: "PUT",
+  return fetch(`${karafRootAddr}/templates/register`, {
```

Comment:
Not sure why I did not catch this before but you cannot remove the key from this URI. There is no body to this fetch.